packages/langium/test/lsp/completion-provider.test.ts

```diff
@@ -178,6 +179,39 @@ describe('Completion within alternatives', () => {
         });
     });
 
+    test('Should show documentation on completion items', async () => {
+        const grammar = `
+        grammar g
+        entry Model: (elements+=(Person | Greeting))*;
+        Person: 'person' name=ID;
+        Greeting: 'hello' (person=[Person:ID]);
+        terminal ID: /\\^?[_a-zA-Z][\\w_]*/;
+        hidden terminal WS: /\\s+/;
+        hidden terminal ML_COMMENT: /\\/\\*[\\s\\S]*?\\*\\//;
+        `;
+
+        const services = await createServicesForGrammar({ grammar });
+        const completion = expectCompletion(services);
+        const text = `
+        /** Hello this is A */
+        person A
+        hello <|>
+        `;
+
+        await completion({
+            text,
+            index: 0,
+            expectedItems: ['Hello this is A'],
+            itemToString: item => {
+                if (MarkupContent.is(item.documentation)) {
```

Comment:
For a test it confuses me that you have an if-else here... can it be something different in 2 runs?

Reply:
You could use an assertion instead. A branch that is never called, feels also weird.

Reply:
Right, I've updated the code to use an assertion 👍

Reply:
I meant to assert First without an If. Can show on Monday

Reply:
Something like this here... it does not need an `if`. You just assume that you are in the right branch. If you are not you get an assertion error.

```ts
assert(MarkupContent.is(item.documentation));
return item.documentation.value;
```

In doubt, you can always create your own assert function.

```ts
function myAssert(condition: boolean) : asserts condition {
  if(!condition) throw Error();
}
```

I am not sure if you already know this, so I wrote it down. It makes testing a bit shorter and helps also to make some intrinsic assumptions visible.

Reply:
Thanks, that looks way better 👍